### PR TITLE
UNI-275 and UNI-279

### PR DESCRIPTION
**Pull request by lscheinkman**
@brev @marionleborgne This is the combined PR for both UNI-275 and UNI-279 presented at the demo session.
Let me know if if it is ok to merge.

### CODE REVIEW

File: unicorn/js/browser/actions/LoadMetricData.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// Numenta Platform for Intelligent Computing (NuPIC)
 // Copyright (C) 2015, Numenta, Inc.  Unless you have purchased from
```
Comment (shantanoo) [inline unicorn/js/browser/actions/LoadMetricData.js]: Change `(C)` to `©`

File: unicorn/js/browser/actions/UnloadMetricData.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-// Numenta Platform for Intelligent Computing (NuPIC)
 // Copyright (C) 2015, Numenta, Inc.  Unless you have purchased from
```
Comment (shantanoo) [inline unicorn/js/browser/actions/UnloadMetricData.js]: `s/\(C\)/©/`

File: unicorn/js/browser/app.js [hunk 3]
```diff
@@ -50,6 +50,7 @@ import FileStore from './stores/FileStore';
 import FileDetailsStore from './stores/FileDetailsStore';
 import ModelStore from './stores/ModelStore';
 import ModelDataStore from './stores/ModelDataStore';
+import MetricDataStore from './stores/MetricDataStore';
```
Comment (shantanoo) [inline unicorn/js/browser/app.js]: Sort the imports in ascending order.

File: unicorn/js/browser/lib/Constants.js [hunk 4]
```diff
@@ -57,9 +61,11 @@ export const ACTIONS = Object.freeze({
...
   LIST_METRICS_FAILURE: 'LIST_METRICS_FAILURE',
   LIST_METRICS: 'LIST_METRICS',
-  RECEIVE_DATA: 'RECEIVE_DATA',
-  SEND_DATA_FAILED: 'SEND_DATA_FAILED',
-  SEND_DATA: 'SEND_DATA',
+  RECEIVE_MODEL_DATA: 'RECEIVE_MODEL_DATA',
+  SEND_METRIC_DATA_FAILED: 'SEND_METRIC_DATA_FAILED',
+  SEND_METRIC_DATA: 'SEND_METRIC_DATA',
+  LOAD_METRIC_DATA: 'LOAD_METRIC_DATA',
+  UNLOAD_METRIC_DATA: 'UNLOAD_METRIC_DATA',
```
Comment (shantanoo) [inline unicorn/js/browser/lib/Constants.js]: Please sort the keys in ascending order.

**Comment by brev** [comment]
sorry for the delay, looking now

**Comment by brev** [comment]
thanks for taking the time @lscheinkman this looks great, I see all my changes there. :+1:

**Pull request merged**